RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -380,7 +383,7 @@ extension VoiceBroadcastPlaybackViewModel: VoiceBroadcastAggregatorDelegate {
         
         updateDuration()
         
-        if state.playbackState != .stopped {
+        if state.playbackState != .stopped, (state.playbackState != .paused && isActuallyPaused == true) {
```

Comment:
I'm not sure to understand why you add this condition here
I'm wondering if `isActuallyPaused == true` is not enough

Reply:
I updated this condition in latests commits.